 flathub.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
You don't need an empty `flathub.json`. You can just remove it.

Reply:
Got it. Done.